src/react/hooks/__tests__/useQuery.test.tsx

```diff
@@ -2092,6 +2101,185 @@ describe("useQuery Hook", () => {
       unmount();
       result.current.stopPolling();
     });
+
+    it.only("should stop refetching if `focusPolling: true` at global level", async () => {
```

Comment:
```suggestion
    it("should stop refetching if `focusPolling: true` at global level", async () => {
```

Let's make sure the other tests can execute as well 🙂

Reply:
This was unintentional. I was using this to run specific tests locally only.